libsyclinterface/source/dpctl_sycl_platform_interface.cpp

```diff
@@ -84,120 +87,125 @@ __dpctl_give DPCTLSyclPlatformRef DPCTLPlatform_CreateFromSelector(
         try {
             P = new platform(*DS);
             return wrap(P);
-        } catch (std::bad_alloc const &ba) {
-            std::cerr << ba.what() << '\n';
-            return nullptr;
         } catch (runtime_error const &re) {
             delete P;
-            std::cerr << re.what() << '\n';
+            error_handler(re, __FILE__, __func__, __LINE__);
```

Comment:
``sycl::runtime_error`` is deprecated, and is also an instance of class derived from `std::exception``